packages/rs-drive-abci/src/abci/handler/process_proposal.rs

```diff
@@ -222,7 +223,11 @@ where
             status: proto::response_process_proposal::ProposalStatus::Accept.into(),
             validator_set_update,
             // TODO: Implement consensus param updates
-            consensus_param_updates: None,
+            consensus_param_updates: get_consensus_params_update(
+                &app.platform().config.abci.consensus_params_dir,
+                request.height,
+            )
+            .map_err(|e| Error::Abci(AbciError::ConsensusParams(e.to_string())))?,
```

Comment:
This can not work, because it would require the entire network to update on the same block. Something that is possible for a devnet, but not in a real network. Consensus params imo should be versioned, and can only change if the network votes for it in consensus.